xenforo/library/bdApi/XenForo/Model/Alert.php

```diff
@@ -34,8 +34,7 @@ public function bdApi_getAlertsByIds($alertIds)
 					user.gender, user.avatar_date, user.gravatar, user.username
 				FROM xf_user_alert AS alert
 				INNER JOIN xf_user AS user ON (user.user_id = alert.user_id)
-				WHERE alert.view_date = 0
-					AND alert.alert_id IN (' . $this->_getDb()->quote($alertIds) . ')
+				WHERE alert.alert_id IN (' . $this->_getDb()->quote($alertIds) . ')
```

Comment:
Sửa line này hơi rủi ro do các chỗ khác khi gọi hàm này nó đang hiểu là không trả về alert đã đọc -> cần rà lại call sites. Nếu an toàn thì sửa được, còn không thì revert lại hoặc thêm param.

Reply:
Đã kiểm tra thì mới dùng 2 chỗ trong add-on bdApi. Với cả thấy chỉ lấy unread alerts ko đúng lắm vì đang cần lấy theo IDs mà.

Reply:
Uh,  cũng hơi kì. Không hiểu sao lại làm thế. Hình như lấy theo ids khi ping phải không? Lúc đó sẽ auto skip nếu alert đã đọc rồi thì phải.

Vậy sửa line này cũng được. Nhưng sửa thêm bên kia để nó tự check đã đọc hay chưa.

Reply:
Bên kia thì check từ lâu rồi anh.

Reply:
Ơ ảo =))

Reply:
Có cái đợt điều chỉnh Ping e có điều chỉnh á :)) chứ ko phải từ đầu  B-)